packages/settings-library/src/settings_library/redis.py

```diff
@@ -18,7 +18,8 @@ class RedisSettings(BaseCustomSettings):
     REDIS_PASSWORD: Optional[SecretStr] = None
 
     # db
-    REDIS_DB: Optional[str] = "0"
+    REDIS_DB: str = "0"
```

Comment:
is there a reason why you dropped the `Optional` here. @pcrespov was it required?

Reply:
yes, because it IS required. if not then it is empty and that is not the intention

Reply:
you mean it is NOT nullable, because this field happens to be optional (it has a default!)

Reply:
yes but if optional then it may return None which is not what I want